support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx

```diff
@@ -88,14 +90,19 @@ const ensureRoundedDoubleDigits = (timeSection: number): string => {
 
 // return the countdown component
 export default function Countdown({
+	show,
+	setShow,
 	countdownCampaign: campaign,
 }: CountdownProps): JSX.Element {
 	// one for each timepart to reduce DOM updates where unnecessary.
 	const [seconds, setSeconds] = useState<string>(initialTimePart);
 	const [minutes, setMinutes] = useState<string>(initialTimePart);
 	const [hours, setHours] = useState<string>(initialTimePart);
 	const [days, setDays] = useState<string>(initialTimePart);
-	const [showCountdown, setShowCountdown] = useState<boolean>(false);
+
+	const hideMyself = () => {
```

Comment:
Could this be renamed to something a bit clearer? Maybe something like `hideCountdown`

Reply:
I've changed that, thanks!